### ISSUE TITLE
Go to pkg.go.dev

### ISSUE THREAD

**Issue by gertcuykens** [golang/vscode-go#663]
https://github.com/golang/go/issues/41488 (made the mistake thinking this can't be done without gopls modification so I copy it here)

Hover space is precious, only the most necessary description should be in it

![image](https://user-images.githubusercontent.com/863731/93644169-bc57bf80-fa01-11ea-9a43-63682311faa9.png)

`http.HandlerFunc on pkg.go.dev`

is not the place for it, it should be in the right click navigation menu `Go to pkg.go.dev` instead

![image](https://user-images.githubusercontent.com/863731/93644354-1bb5cf80-fa02-11ea-9df3-31e5e05cd015.png)

vscode has a special context menu for it called navigation for all navigation actions like  `Go to pkg.go.dev`

![image](https://user-images.githubusercontent.com/863731/93649862-3fcade00-fa0d-11ea-82ea-caafe5096d4b.png)

**Comment by hyangah** [comment]
Thanks for filing the issue. Users can disable the gopls feature with the latest gopls.

For registering it in the explorer context menu, contribution is welcome!

As you mentioned https://code.visualstudio.com/api/references/contribution-points#contributes.menus provides the info on how to add to an entry. Indeed the navigation group seems like a good fit.

You will need to implement a command - the command retrieves the position and computes the link to the document server of user's choice. It needs some research on LSP.
Initially I thought LSP's `textDocument/documentLink` request may be useful, but that is not taking a position inside a doc so not suitable. `documentLink/resolve` seems promising but we need to check whether gopls supports it.

**Comment by Kunamatata** [comment]
I disagree with the link not being in hover. Adding the possibility to add a context menu item an removing the link is fine, but the link should be there by default. It doesn't use much space and it is very valuable to have a quick link to the documentation.

**Comment by ant1m** [comment]
`documentlink/resolve` is not implemented in gopls. The link to documentation (linkPath) is, AFAIK, only found in the `textDocument/hover` response, and should be scraped from there. The linkPath in the hover response is a gopls configuration option and can be toggled, so there is no guaranty that it can always be found.
